Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.py

```diff
@@ -1274,8 +1350,267 @@ def test_function(client: MsGraphClient, args, has_access_to_context=False):
                      f'Please check authentication related parameters. [{response.status_code}]')
 
 
+def get_message_user(client, message_user):
+    is_email = re.search(EMAIL_REGEX, message_user)
+    if is_email:
+        user_result = (client.get_user_id(message_user)).get('value')
+        if not user_result:
+            raise DemistoException(f'{message_user} is not a valid user')
+        return user_result[0].get('id')
```

Comment:
is it possible for user_result not to be None and not to have id ? causing an exception

Reply:
if value is not empty, its a list of user's dict, which in each dict there's an id for sure for the looked up user